python/nav/web/netmap/graph.py

```diff
@@ -59,13 +51,8 @@ def get_topology_graph(layer=2, load_traffic=False, view=None):
         return _json_layer3(load_traffic, view)
 
 
+@partial(cache_topology, "layer 2")
```

Comment:
Using partial here kind of obscures the decorator's intention. It's better to use a two-level decorator implementation when the decorator has required arguments. [Here's one example from NAV code that does this.](https://github.com/UNINETT/nav/blob/a60648e357c1f666e0c1f39aeea643fd19ce093f/python/nav/web/alertprofiles/decorators.py#L26)